android-components/components/service/firefox-accounts/src/main/java/mozilla/components/service/fxa/AccountStorage.kt

```diff
@@ -190,6 +196,10 @@ internal abstract class AbnormalAccountStorageEvent : Exception() {
      * Account state was expected to be present, but it wasn't.
      */
     internal class UnexpectedlyMissingAccountState : AbnormalAccountStorageEvent()
+    internal class RestoringNonConnectedAccount(val state: FxaRustAuthState) : AbnormalAccountStorageEvent() {
+        override val message: String
+            get() = "AbnormalAccountStorageEvent.RestoringNonConnectedAccount($state)"
+    }
```

Comment:
@mavduevskiy Is this the right way to get message strings included in the Sentry report?

Reply:
It is certainly a way to get it there! Though one could argue that a more canonical approach for Java/Kotlin throwables would be passing a message into the constructor. We also don't really use `state` as far as I could tell, so 

```
internal class RestoringNonConnectedAccount(override val message: String) : AbnormalAccountStorageEvent()
```
used as
```
crashReporter?.submitCaughtException(
                    AbnormalAccountStorageEvent.RestoringNonConnectedAccount(
                        "AbnormalAccountStorageEvent.RestoringNonConnectedAccount($state)",
                    )
                )
```
could be a way to go.

Reply:
I swear I saw a message from you about this, but I don't see it anymore.  In any case, I changed this to use pass a `message` parameter to the exception constructor instead.